docs/SMG/08-deployment.md

```diff
@@ -138,6 +138,11 @@ docker-compose build
 docker-compose up --detach
 ```
 
+### Running database migrations
+If any migrations need to be run on the database (in the `src/API/src/db/migrations` folder), run the following ocmmand to apply these migrations to the db:
+```
+docker run -it -e POSTGRES_USER=$VA_DB_USER -e POSTGRES_PASSWORD=$VA_DB_PASSWORD -e POSTGRES_HOST=vectoratlas-db.postgres.database.azure.com -e POSTGRES_DB=postgres -e DB_ENCRYPT_CONNECTION=true docker-api npm run migrations:runallprod
+```
```

Comment:
This should work for now as the API will keep restarting whilst the data model doesn't update. Eventually we can include this in an update script.